scripts/stg-env.js

```diff
@@ -6,8 +6,8 @@ fs.readFile(envConfig, 'utf8', function (err, data) {
         return console.log(err);
     }
     const result = data
-        .replace('cacheServerUrl = \'https://identitycache-dev.energyweb.org/v1\'', 'cacheServerUrl = \'https://volta-identitycache.energyweb.org/v1\'')
-        .replace('natsServerUrl = \'https://identityevents-dev.energyweb.org/\'', 'natsServerUrl = \'https://volta-identityevents.energyweb.org/\'')
+        .replace('cacheServerUrl = \'https://identitycache-dev.energyweb.org/v1\'', 'cacheServerUrl = \'https://identitycache-staging.energyweb.org//v1\'')
```

Comment:
Is that additional slash correct?
```suggestion
        .replace('cacheServerUrl = \'https://identitycache-dev.energyweb.org/v1\'', 'cacheServerUrl = \'https://identitycache-staging.energyweb.org/v1\'')
```

Reply:
probably not. I will remove it.